core/arch/arm/plat-k3/conf.mk

```diff
@@ -1,6 +1,7 @@
 CFG_WITH_STATS ?= y
 CFG_CRYPTO_WITH_CE ?= y
 CFG_CONSOLE_UART ?= 0
+CFG_CONSOLE_RUNTIME ?= y
```

Comment:
This config switch name looks very generic. It should be documented in _mk/config.mk_ or be renamed to something like `CFG_K3_CONSOLE_RUNTIME`.

Maybe a generic switch would set a specific level post boot, like `CFG_CORE_RUNTIME_LOG_LEVEL`.

Reply:
> Maybe a generic switch would set a specific level post boot, like `CFG_CORE_RUNTIME_LOG_LEVEL`.

+1. Note that it would waste some memory space, for example if `CFG_TEE_CORE_LOG_LEVEL=3` and the post-boot log level is 2 then many debug messages (level 3) would be there but never used.

Reply:
True. 
Thinking about this, would be make sense to use for example the stats PTA or device PTA to modify the log level at runtime?

Reply:
I like the idea although it is probably a different use case than what this PR is supposed to address.